pkg/config/config_test.go

```diff
@@ -0,0 +1,84 @@
+package config
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Configuration", func() {
+	Context("ReadConfig", func() {
+		It("Read configuration from environment variables", func() {
+			dc := &DaemonConfig{}
+
+			Expect(os.Setenv("PLUGIN", "ufm")).ToNot(HaveOccurred())
+			Expect(os.Setenv("PERIODIC_UPDATE", "10")).ToNot(HaveOccurred())
+			Expect(os.Setenv("GUID_RANGE_START", "02:00:00:00:00:00:00:00")).ToNot(HaveOccurred())
+			Expect(os.Setenv("GUID_RANGE_END", "02:00:00:00:00:00:00:FF")).ToNot(HaveOccurred())
+			Expect(os.Setenv("UFM_USERNAME", "admin")).ToNot(HaveOccurred())
+			Expect(os.Setenv("UFM_PASSWORD", "123456")).ToNot(HaveOccurred())
```

Comment:
general question : are these cleaned between tests ? in the future it may affect other test cases

Reply:
i recommend to add and AfterEach() clause to clean up env variables

Reply:
added after AfterEach()